helpers/class.ExtensionHelper.php

```diff
@@ -159,23 +161,6 @@ public static function checkRequiredExtensions(common_ext_Extension $extension)
                     'GENERIS'
                 );
             }
-            if ($requiredVersion != '*') {
```

Comment:
Sorry, I'm not familiar with this part of the code. Could you please detail what as the purpose of this original code and why it's not required anymore ?

Reply:
This block of code was used to check if version of extension corresponds to the restrictions defined in the manifest (in case if * (any version) this check is skipped).
But after this PR will be merged all the dependency/version management will be done by composer so there is no need to check it in Tao anymore.